.github/workflows/pull-request.yml

```diff
@@ -10,3 +10,17 @@ jobs:
       with:
         configuration-path: .github/labeler-pull-request-triage.yml
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
+    # See also: https://github.com/CodelyTV/pr-size-labeler/pull/26
+    - uses: bflad/pr-size-labeler@7df62b12a176513631973abfe151d2b6213c3f12
```

Comment:
I think the security RFC indicated that we should fork actions into the HashiCorp org, didn't it?

Reply:
In SEC-047 we've opted to pin via commit SHA over forking due to maintenance burden:

> For third-party GitHub Actions with broad adoption across the organization, the repository may _optionally_ be forked into the hashicorp GitHub Organization. All recommendations for HashiCorp-maintained actions should be implemented on the forked repository. (The cost of managing/maintaining a fork should be considered, and pinning to an external trusted commit SHA is a reasonable alternative.)